### PR TITLE
Replace crossbow infinity with vanilla infinity

### LINKED ISSUE #685: Replace vampirism infinity enchanment
Remove the vampirism infinity enchantment and use the vanilla one using `Item#canApply` and `Item#canApplyAtEnchantmenttable`

While we are at it, also look into arrows, are there vanilla crossbow arrows now, do the vanilla arrows have to be different from the Vampirism crossbow ones?

### PR DESCRIPTION
**Pull request by maxanier**
TODO:
- [x] Test

Closes #685

### CODE REVIEW

File: src/main/resources/data/vampirism/recipes/weapontable/crossbow_arrow_from_vanilla.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  "result": {
+    "item": "vampirism:crossbow_arrow",
+    "count": 1
+  },
+  "level": 1,
+  "lava": 0,
+  "type": "vampirism:shapeless_crafting_weapontable",
+  "ingredients": [
+    {
+      "item": "minecraft_arrow"
```
Comment (Cheaterpaul) [inline src/main/resources/data/vampirism/recipes/weapontable/crossbow_arrow_from_vanilla.json]: minecraft:arrow

**Review by Cheaterpaul — DISMISSED** [review]
The crossbow can fire even without arrows in the inventory. Not sure if this is intended/was not a problem before.

**Comment by maxanier** [comment]
This is vanilla behavior apparently.

**Comment by Cheaterpaul** [comment]
because with the bow this does not happen. Make sure to be not in creative mode.

**Comment by maxanier** [comment]
ah, damn creative mode

**Pull request merged**